src/io/flutter/logging/FlutterLogEntry.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.logging;
+
+import com.intellij.openapi.util.text.StringUtil;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class FlutterLogEntry {
+
+  // TODO(pq): assuming we keep a timestamp, consider making it a long.
+  @NotNull
+  private final String timestamp;
```

Comment:
I think having timestamps for the log entries is potentially useful, even if we son't display them. Just for comparison purposes and other, this should likely not be already converted to user facing text, but be in some more semantic type, like a `long` / `Date` / ...

Reply:
Sounds good.  Agreed!